modules/ROOT/pages/dataweave-formats-avro.adoc

```diff
@@ -99,36 +103,38 @@ output application/avro schemaUrl="classpath://schema.json"
 ----
 
 
-// CONFIG PROPS ///////////////////////////////////////////////////////
-
 [[properties]]
 == Configuration Properties
 
-DataWeave supports the following configuration properties for Avro.
+DataWeave supports the following configuration properties for the Avro format.
 
-=== Reader Properties (for Avro)
+
+[[reader_properties]]
+=== Reader Properties
```

Comment:
TODO: "THE <format_name> format" --> "This format" everywhere for same reasons as above